src/core/lib/event_engine/cf_engine/cf_engine.cc

```diff
@@ -89,12 +109,12 @@ std::unique_ptr<EventEngine::DNSResolver> CFEventEngine::GetDNSResolver(
   grpc_core::Crash("unimplemented");
 }
 
-void CFEventEngine::Run(EventEngine::Closure* /* closure */) {
-  grpc_core::Crash("unimplemented");
+void CFEventEngine::Run(EventEngine::Closure* closure) {
+  executor_->Run(closure);
```

Comment:
there's a null check here, do we need it?
https://github.com/grpc/grpc/blob/1bcaa3e9351a9b00f5ce2a4b68c5e3bf61f6be8d/src/core/lib/event_engine/posix_engine/posix_engine.cc#L301-L312

Reply:
I can add the check but it won't help anything as
1. unlike that in PosixEnginePollerManager, here executor_ is created in engine constructor so will not be null 
2. if it were null then closures cannot run so it will stuck which is not better than crash where we have stack trace
3. can we instead somehow annotate it not null or assert here?

Reply:
That's a fine justification. Thanks.